src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -202,6 +202,13 @@ public TagHelperContent SetContent(IHtmlContent htmlContent)
         /// <returns>A <see cref="string"/> containing the content.</returns>
         public abstract string GetContent();
 
+        /// <summary>
+        /// Gets the content.
+        /// </summary>
+        /// <param name="encoder">The <see cref="IHtmlEncoder"/>.</param>
+        /// <returns>A <see cref="string"/> containing the content.</returns>
+        public abstract string GetContent(IHtmlEncoder encoder);
```

Comment:
We only expose the abstract class to users when they do things like `output.Content`; therefore it needs to be rich enough to work with. 

Maybe a better alternative would be for this version to be the default and have a `GetContent()` extension method?